bin/namespaced-wireguard-vpn-interface

```diff
@@ -26,7 +26,10 @@ case "$1" in
 
         # Add default routes for IPv4 and IPv6
         ip -n "$NETNS_NAME" -4 route add default dev "$WIREGUARD_NAME" || die
-        ip -n "$NETNS_NAME" -6 route add default dev "$WIREGUARD_NAME" || die
+        if ip -o -6 -a | grep "${wg_names[$i]}"
```

Comment:
I think this needs to be `$WIREGUARD_NAME` now. Also, we should use `grep -q` to avoid unnecessary output.